tools/src/main/java/org/apache/kafka/tools/reassign/ReassignPartitionsCommandOptions.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kafka.tools.reassign;
+
+import joptsimple.ArgumentAcceptingOptionSpec;
+import joptsimple.OptionSpecBuilder;
+import org.apache.kafka.server.util.CommandDefaultOptions;
+
+public class ReassignPartitionsCommandOptions extends CommandDefaultOptions {
+    // Actions
+    private final OptionSpecBuilder verifyOpt;
+    private final OptionSpecBuilder generateOpt;
+    private final OptionSpecBuilder executeOpt;
+    private final OptionSpecBuilder cancelOpt;
+    private final OptionSpecBuilder listOpt;
+
+    // Arguments
+    private final ArgumentAcceptingOptionSpec<String> bootstrapServerOpt;
```

Comment:
For consistency with the other tools, I would use the `OptionSpec` superclass.

Reply:
Thanks. fixed.